web-api/src/applicationContext.ts

```diff
@@ -99,7 +99,7 @@ const entitiesByName = {
 };
 
 export const createApplicationContext = (
-  appContextUser,
+  appContextUser = {},
```

Comment:
Not sure if this is important, but why are we defaulting this? It seems like we would want this to be undefined for the conditional on 107